### PR TITLE
Fixes hyperlinks to external resources in docs for MDAnalysis.coordinates.PQR

### LINKED ISSUE #1826: links in doc to PQR pages are broken
### Expected behaviour
Outgoing links on https://www.mdanalysis.org/mdanalysis/documentation_pages/coordinates/PQR.html should be pointing somewhere.

### Actual behaviour
Broken links for PQR.

### Fix
Reason: http://www.poissonboltzmann.org/ was reorganized (can also use SSL https://www.poissonboltzmann.org/)

The fileformat is now at:
* https://apbs-pdb2pqr.readthedocs.io/en/latest/formats/pqr.html

### PR DESCRIPTION
**Pull request by palnabarun**
Fixes #1826

Changes made in this Pull Request:
 - Updates stale broken links

PR Checklist
------------
 - [ ] Tests?
 - [x] Docs?
 - [ ] CHANGELOG updated?
 - [x] Issue raised/referenced?

### CODE REVIEW

File: package/MDAnalysis/coordinates/PQR.py [hunk 1]
```diff
@@ -92,10 +92,10 @@
 
    pdb2pqr --ff=charmm --whitespace 4ake.pdb 4ake.pqr
 
-.. _PQR:     http://www.poissonboltzmann.org/file-formats/biomolecular-structurw/pqr
-.. _APBS:    http://www.poissonboltzmann.org/apbs
-.. _PDB2PQR: http://www.poissonboltzmann.org/pdb2pqr
-.. _PDB:     http://www.rcsb.org/pdb/info.html#File_Formats_and_Standards
+.. _PQR:     https://apbs-pdb2pqr.readthedocs.io/en/latest/formats/pqr.html
```
Comment (orbeckst) [inline package/MDAnalysis/coordinates/PQR.py]: According to https://travis-ci.org/MDAnalysis/mdanalysis/jobs/354516290, the target PQR is defined elsewhere, too, and the doc build fails. Please find and fix.

Comment (palnabarun) [reply package/MDAnalysis/coordinates/PQR.py]: I found that and replaced it.

Comment (orbeckst) [reply package/MDAnalysis/coordinates/PQR.py]: Thanks!

**Comment by orbeckst** [comment]
Thanks @palnabarun – squashed and merged.

**Comment by palnabarun** [comment]
Welcome @orbeckst

**Pull request merged**